purchases/src/test/java/com/revenuecat/purchases/PurchasesTest.kt

```diff
@@ -113,12 +117,13 @@ class PurchasesTest {
 
     private val stubOfferingIdentifier = "offering_a"
     private val stubProductIdentifier = "monthly_freetrial"
+    private val stubProductGroupIdentifier = "monthly_freetrial"
     private val oneOfferingsResponse = "{'offerings': [" +
         "{'identifier': '$stubOfferingIdentifier', " +
         "'description': 'This is the base offering', " +
         "'packages': [" +
-        "{'identifier': '\$rc_monthly','platform_product_identifier': '$stubProductIdentifier'}" +
-        "]}]," +
+        "{'identifier': '\$rc_monthly','platform_product_identifier': '$stubProductIdentifier'," +
+        "'platform_product_group_identifier': '$stubProductGroupIdentifier'}]}]," +
```

Comment:
This JSON is missing `,"product_duration":"P1M"`

Reply:
Ah yeah, I was checking out a test and added this and forgot to remove it. But I will add that instead 👍